### ISSUE TITLE
IntellIJ warning as "Deprecated API"

### ISSUE THREAD

**Issue by 11Ramos11** [TheoKanning/openai-java#130]
This is the dependency I'm using on my gradle:

`implementation 'com.theokanning.openai-gpt3-java:client:0.10.0'`

IntellIJ is giving a warning of "Class OpenAI uses or overrides a deprecated API".

Not only this, but it has been exhibiting weird behaviors such as this prompt/answer:

Prompt:
> how are you

Answer:
> ?
> 
> I'm good. How are you?
> 
> I'm good.
> 

This is weird, 2/10 times, it gives a normal response, but it's been this weird.

My OpenAIClass:

```java
package com.package.name;

import com.theokanning.openai.OpenAiService;
import com.theokanning.openai.completion.CompletionRequest;
import com.theokanning.openai.image.CreateImageRequest;

public class OpenAI {

    private static final String API_KEY = "censored";
    private static OpenAI openAI = null;
    private OpenAiService service;

    private OpenAI(){
        service = new OpenAiService(API_KEY);
    }

    public static OpenAI getInstance(){

        if (openAI == null)
            openAI = new OpenAI();
        return openAI;
    }
    public String request(String message) {

        try {
            CompletionRequest request = CompletionRequest.builder()
                    .model("text-davinci-002")
                    .maxTokens(500)
                    .prompt(message)
                    .temperature(0.1)
                    .build();

            String response = service.createCompletion(request).getChoices().get(0).getText();

            return response;
        } catch (Exception e) {
            return "Something went wrong!";
        }
    }
}
```

**Comment by cryptoapebot** [comment]
Couple of things. 

Add a timeout to your service. 
```
        service = new OpenAiService(API_KEY, Duration.ofSeconds(45));
```

Switch to the newer davinci model.
```
                    .model("text-davinci-003")
```

You probably want to add stops to your submission.  Change to fit your interaction.  "How are you?\n"  with a question mark newline as a stop.  Or, "Answer:\n"
```
	public static final List<String> stops = Arrays.asList("\n", " Human:", " AI:");

					.stop(stops)
```

Finally, maybe play w/ the prompt by giving it a few more examples of what's being requested, and play w/ the other params. 
```
					.temperature(0.97)
					.topP(1.0)
					.frequencyPenalty(0.0)
					.presencePenalty(0.0)
```

**Comment by 11Ramos11** [comment]
Hi, thanks for the input! It got better, but there are still some weird interactions going on.

Config:

```java

CompletionRequest request = CompletionRequest.builder()
                    .model("text-davinci-003")
                    .maxTokens(2048)
                    .prompt(message)
                    .temperature(0.01)
                    .topP(1.0)
                    .frequencyPenalty(0.5)
                    .presencePenalty(0.0)
                    .stop(stops)
                    .build();
```

I am using the stops() you suggested as well. 

But here is an example of what's going on:
                 
![image](https://user-images.githubusercontent.com/92692460/222555759-19b1119b-559c-445c-a8f6-d3b9385388dc.png)

**Comment by cryptoapebot** [comment]
```
        List<CompletionChoice> choices = service.createCompletion(completionRequest).getChoices();
        assertEquals(5, choices.size());

      for (CompletionChoice c : choices) {
          System.out.println(c.getText()); // or whatever the exact command is. 
      }
```

**Comment by 11Ramos11** [comment]
Hi, thanks for the help. I have concluded why I was having issues after playing with the playground, like you suggested. 

Turns out that the AI is set to complete mode, hence why this happens:

![image](https://user-images.githubusercontent.com/92692460/223727083-fe304e4d-ac43-467a-abda-91aeb8e8671e.png)

The ", I'm not a Doctor" is him completing the prompt, and then the rest is the actual reply to said prompt.

I am able to turn this off in the playground, but I have no idea how to do it within the Java API. 
Any tips?

P.S.: Giving him context of being a helpful discord bot was GREAT! Thank you!!
